french/dessine/miel/ouvrir-porte.md

```diff
@@ -0,0 +1,6 @@
+Vous ouvrez la porte en miel mais, telle une mouche sur du papier tue-mouche, votre main reste collée à la poignée...
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.